modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -10,7 +10,10 @@ import Scalaz._
 case class QState(pending: Queue[Action],
                   current: Current,
                   done: Queue[Result],
-                  status: Status)
+                  status: Status) {
+
+  def isEmpty: Boolean = this.pending.sequences.isEmpty && this.current.isEmpty
```

Comment:
Never mind my previous comment. I just realized the parameters of a case class are in scope in its case class block.